src/app/shared/team-radio-switch/team-radio-switch.model.ts

```diff
@@ -0,0 +1,7 @@
+export class Team {
```

Comment:
Consider using TS Interfaces for this purpose

Reply:
done